### ISSUE TITLE
antd-mobile.less to be processed into PostCSS plugins, but not want to be processed into CSS Modules!

### ISSUE THREAD

**Issue by fangjj** [nathantreid/meteor-css-modules#120]
i want node_modules/antd-mobile/dist/antd-mobile.less to be proccessed into postcss-pxtorem plugin but not want to be proccessed into css-modules,  this is rep https://github.com/fangjj/test-css-module
if i config  passthroughPaths: [ "client/antd.less" ] then node_modules/antd-mobile/dist/antd-mobile.less will not be proccessed into postcss-pxtorem plugin.

**Comment by fangjj** [comment]
any update?

**Comment by akanix42** [comment]
This seems like a reasonable feature to include, but I'm not sure when I'll be able to do so. I'm open to PR's as well. In the meantime, the best solution I can think of is to add `juliancwirko:postcss` (see instructions here: https://github.com/juliancwirko/meteor-postcss/)  which will apply postcss to your files after `nathantreid:css-modules` has done it's part. I haven't tested it with my plugin, but I think they should be compatible.

**Comment by fangjj** [comment]
thank you! i add juliancwirko:postcss to proccessed what CSS Modules passthroughPaths. like this.
![image](https://user-images.githubusercontent.com/15318800/41631803-3a660fce-7469-11e8-8671-efcea42cf1ac.png) it works for me. but i have a question for this. 
![image](https://user-images.githubusercontent.com/15318800/41631845-7f27b5b8-7469-11e8-8012-e8bbcd3b24e3.png)

**Comment by akanix42** [comment]
Yeah, those are the packages that actually perform the postcss CSS modules
processing.

See here for more info:
https://github.com/nathantreid/meteor-css-modules/blob/master/README.md#postcss-plugins

**Comment by fangjj** [comment]
yeah, got it. when  remove this required, but it still work.
